articles/api-auth/grant/hybrid.md

```diff
@@ -0,0 +1,75 @@
+  - implicit
+contentType: concept
+useCase:
+  - secure-api
+  - call-api
+---
+# Call APIs Using the Hybrid Flow
+
+The Hybrid Flow is an OpenID Connect (OIDC) flow that draws from the following:
+
+1. [Authorization Code Grant](/api-auth/grant/authorization-code)
+2. [Implicit Grant](/articles/api-auth/grant/implicit)
+
+The Hybrid Flow enables use cases where your application can immediately use an ID token to access information about the user while obtaining an authorization code that can be exchanged for an Access Token (therefore gaining access to protected resources for an extended period of time).
+
+## Background
+
+With the [Authorization Code Grant](/api-auth/grant/authorization-code), Auth0 sends you an authorization code (authorization not required), which your app then sends in to retrieve tokens (authorization required). Your application authorizes itself with a Client ID and Client Secret stored securely on your server.
+
+On the other hand, the [Implicit Grant](/articles/api-auth/grant/implicit) allows you to request Access Tokens without needing to authenticate your application. Auth0 verifies your app's identity based on the provided redirect URI. Because of this, you shouldn't utilize long-lived Access Tokens, and you cannot use Refresh Tokens.
+
+## The Hybrid Flow
+
+The Hybrid Flow allows you to take advantage of aspects of both the Authorization Code and Implicit Grants. You can request any combination of Access Tokens, ID Tokens, and authorization codes from either a server-side web application or a single page application (SPA).
+
+For each interaction with Auth0, you will receive two (sometimes three) items in response:
+
+1. An authorization code and an Access Token
+1. An authorization code and an ID Token
+1. An authorization code, an Access Token, and an ID Token
+
+In this article, we will take a closer look at how this flow works.
+
+## Benefits of the Hybrid Flow
+
+The benefits of using the Hybrid Flow include the following:
+
+* Can get tokens from the authorization *and* token endpoints
+* Can send tokens via the user
+* Can authorize the application
+* Can use refresh tokens
+
+## Overview of the flow
+
+1. The web application initiates the authorization flow and redirects the browser to Auth0 (specifically, the [Authorization Endpoint](/api/authentication#authorization-code-grant)) so that the user can authenticate.
+
+1. Auth0 authenticates the user via the browser. If this is the first time the user does this, they will see a consent page listing the permissions that Auth0 will give to the application.
+
+1. Auth0 redirects the user to the app's **redirect URI** (as specified in the [request to the Authorization endpoint](/api/authentication#authorization-code-grant)) with an Authorization Code included in the querystring (the specific parameter is **code**). Depending on the **response type** indicated in the authorization request, one or more additional parameters may accompany the Authorization Code.
```

Comment:
hybrid responses are by default returned in a fragment. That is unless another response_mode was requested (e.g. form_post or web_message), but very strictly NOT query - that one's forbidden